service/src/main/java/aero/minova/cas/service/ViewService.java

```diff
@@ -132,16 +135,7 @@ public PreparedStatement fillPreparedViewString(Table inputTable, CallableStatem
 				if (iVal != null) {
 					val rule = iVal.getRule();
 					String stringValue = iVal.getValue() + "";
-					if (rule == null) {
-						if (!stringValue.trim().isEmpty()) {
-							sb.append(" ; Position: " + (i + parameterOffset) + ", Value:" + stringValue);
-							preparedStatement.setString(i + parameterOffset, stringValue);
-						} else {
-							// i tickt immer eins hoch, selbst wenn ein Value den Wert 'null', '' hat
-							// damit die Position beim Einfügen also stimmt, muss parameterOffset um 1 verringert werden
-							parameterOffset--;
-						}
-					} else if (rule.contains("in")) {
+					if (rule != null && rule.contains("in")) {
 						List<String> inBetweenValues;
 						inBetweenValues = Stream.of(iVal.getStringValue().split(","))//
```

Comment:
Würde es funktionieren, dass bei rule=between oder rule= in immer String ist in Porstgres?

Reply:
Wahrscheinlich nicht... aber da das im WFC nicht implementiert ist, wird es auch in nächster Zeit nicht vorkommen

Reply:
> Wahrscheinlich nicht... aber da das im WFC nicht implementiert ist, wird es auch in nächster Zeit nicht vorkommen

Vll nicht vom WFC selbt, aber Extemsions könnten es auch benutzen wollen.
Ich würde vorschlagen, das auch dafür zu implementieren. Sollte ja nicht viel sein.

Reply:
Geht nicht, da das iVal in diesen Fällen immer DataType String hat. Damit ist eine Unterscheidung nach dem Typen nicht möglich.